themes/medius/scss/components/links.scss

```
@@ -0,0 +1,235 @@
+.link {
+  @include ff-sans($weight-bold);
+
+  font-size: 16px;
+  line-height: 2;
+  display: inline-block;
+  position: relative;
+  padding-left: 4px;
+  padding-right: 12px;
+  text-decoration: none;
+  transition: all 0.2s ease;
+  z-index: 0;
+  color: $yellow-10;
+  vertical-align: middle;
+
+  &:before {
+    content: "";
+    position: absolute;
+    display: block;
+    height: 2px;
+    right: 8px;
+    bottom: 0;
+    left: 0;
+    z-index: -1;
+    transition: all 0.2s ease;
+    border-radius: 4px;
+    background-color: $coral-60;
+  }
+
+  &:hover,
+  &:focus {
+    color: $yellow-10;
+    padding-left: 8px;
+    padding-right: 8px;
+    border-radius: 4px;
+    text-decoration: none;
+
+    &:before {
+      height: 100%;
+      right: 0;
+    }
+  }
+
+  &:focus {
+    outline: none;
+
+    &:after {
+      content: "";
+      position: absolute;
+      display: block;
+      left: 0;
+      top: 0;
+      right: 0px;
+      bottom: 0;
+      border-radius: 4px;
+      box-shadow: 0 0 0 2px $coral-60 inset, 0 0 0 3px $yellow-10 inset;
+    }
+  }
+
+  @media (min-width: $tablets) {
+    font-size: 20px;
+    line-height: 1.4;
+    padding: 4px 12px 4px 4px;
+  }
+}
+
+.link-swipe-right {
+  @include ff-sans($weight-bold);
+
+  font-size: 16px;
+  line-height: 2;
+  display: inline-block;
+  position: relative;
+  padding-left: 12px;
+  padding-right: 12px;
```

```suggestion
  padding-right: 0;
  max-width: 206px;
  width: 100%;
```

This link has width size of 222px.
Other suggestion is increase the padding-right enough to have ,in total, width of 222px